storage/inmem.go

```diff
@@ -160,9 +163,18 @@ func (i *InMem) AddVacation(ID string, userID string, startDate int64, durationH
 	}
 
 	v.ID = ID
+	v.UserID = userID
 	v.StartDate = startDate
 	v.DurationHours = durationHours
-	employee.Vacations = append(employee.Vacations, v)
+	employee.Vacations = append(employee.Vacations, v.ID)
 	i.employees[userID] = employee
 	return v, nil
 }
+
+func (i *InMem) Vacations() ([]entities.Vacation, error) {
```

Comment:
Is this just the `copy()` built-in function?

Reply:
Basically it is writing of map values to slice.
The map is used to ease operations with vacations by id.

Reply:
oh yes sorry I misread that one.